torch/export/exported_program.py

```diff
@@ -1044,6 +1049,9 @@ class ExportedProgram:
     _verifiers: list[type[Verifier]]
     """List of verifier classes used to validate the exported program."""
 
+    _autocast_state: AutocastState
```

Comment:
@pianpwk Could we make this a generic structure called "Guards" and dump all the future things there?

Reply:
Also I wouldn't only do autocast state. Just do all the global state as a whole here.